### ISSUE TITLE
Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by Catgroove** [sveltejs/svelte-preprocess#284]
**Describe the bug**
When using the svelte-preprocess in rollup, all I get is the error below. The problem goes away if I remove the preprocess altogether from my rollup config.

```
Cannot read property 'length' of undefined
node:internal/modules/cjs/loader:922
  throw err;
  ^

Error: Cannot find module '/Users/martinnorberg/projects/test-snowstorm/__sapper__/dev/server/server.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:15)
    at Function.Module._load (node:internal/modules/cjs/loader:763:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

This is the preprocess config:
```
const preprocess = sveltePreprocess({
  postcss: {
    plugins: [require("postcss-import")],
  },
});
```

It seems to be related to the `postcss` field since removing it fixes the issue.

I assume this is a local issue to my machine but I've not managed to find any solutions. Cheers.

**To Reproduce**
Setup a project according to: https://codechips.me/sapper-with-postcss-and-tailwind/

**Information about your project:**

- Your browser and the version: (e.x. Chrome 52.1, Firefox 48.0, IE 10): n/a

- Your operating system: (e.x. OS X 10, Ubuntu Linux 19.10, Windows XP, etc): OSX 10

- `svelte-preprocess` version: ^4.6.1

- Whether your project uses Webpack or Rollup: rollup

**Additional context**

**Comment by AaronCQL** [comment]
I have this similar issue using `v4.6.1`. Downgrading to and using `v4.5.2` will likely solve this issue.

**Comment by rixo** [comment]
Found this because of this [SO question](https://stackoverflow.com/q/65052081/1387519).

I did a bit of debugging. This happens only in Sapper, with `svelte@3.30`. Downgrading Svelte to `3.29` fixes the issue.

This doesn't happen in a "raw" (non Sapper) Svelte project, it works well with `svelte@3.30` there.

This is most probably related to `svelte@3.30` adding support of sourcemaps for preprocessors. The line that crashes in the compiler is this:

```
if (map.mappings.length == 0)
```

**Comment by Catgroove** [comment]
> Found this because of this [SO question](https://stackoverflow.com/q/65052081/1387519).
> 
> I did a bit of debugging. This happens only in Sapper, with `svelte@3.30`. Downgrading Svelte to `3.29` fixes the issue.
> 
> This doesn't happen in a "raw" (non Sapper) Svelte project, it works well with `svelte@3.30` there.
> 
> This is most probably related to `svelte@3.30` adding support of sourcemaps for preprocessors. The line that crashes in the compiler is this:
> 
> ```
> if (map.mappings.length == 0)
> ```

Worked perfectly. Thanks.

**Comment by Maggi64** [comment]
Should be fixed in svelte 3.31.0
